From 349228e1f2508b21db1ac88066636131dd4cafb4 Mon Sep 17 00:00:00 2001
From: Richard Poettler <richard.poettler@gmail.com>
Date: Tue, 16 Apr 2019 10:28:38 +0200
Subject: [PATCH] colors

---
 config.def.h | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/config.def.h b/config.def.h
index 9373664..1e77f17 100644
--- a/config.def.h
+++ b/config.def.h
@@ -109,6 +109,8 @@ static const char *colorname[] = {
 	/* more colors can be added after 255 to use with DefaultXX */
 	"#cccccc",
 	"#555555",
+	"#cccccc",
+	"#303030",
 };
 
 
@@ -116,8 +118,8 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 7;
-unsigned int defaultbg = 0;
+unsigned int defaultfg = 258;
+unsigned int defaultbg = 259;
 static unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
 
-- 
2.17.2

